### PR TITLE
Style duration field, catch uncaught selects

### PR DESCRIPTION
**Pull request by farridav**
- Ensures we style time fields
- Ensure we convert any plain selects into select2
- Explicitly use dropdownAutoWidth in select2 options

Fixes: https://github.com/farridav/django-jazzmin/issues/166

### CODE REVIEW

File: tests/test_app/polls/models.py [hunk 1]
```diff
@@ -29,7 +29,15 @@ class Choice(models.Model):
     """
 
     poll = models.ForeignKey(Poll, on_delete=models.CASCADE)
-    choice_text = models.CharField(max_length=255)
+    choice_text = models.CharField(
+        max_length=255,
+        choices=(
+            ("Heres a choice", "Heres a choice"),
+            ("Heres another choice", "Heres another choice"),
```
Comment (TysonRV) [inline tests/test_app/polls/models.py:36]: Typos `Here is` 

As it is visible to everyone, I would change them

Comment (farridav) [reply tests/test_app/polls/models.py:36]: hahaha, forever my personal spelling and grammar checker

**Comment by TysonRV** [comment]
Awesome! Looking great. I'll keep an eye on the fields, but this fixes most of them (if not all) 👍

File: jazzmin/static/jazzmin/js/change_form.js [hunk 2]
```diff
@@ -115,7 +115,10 @@ $(document).ready(function () {
...
 
-    if ($tabs.length) {handleTabs($tabs);}
-    else if ($carousel.length) {handleCarousel($carousel);}
-    else if ($collapsible.length) {handleCollapsible($collapsible);}
+    if ($tabs.length) { handleTabs($tabs); }
+    else if ($carousel.length) { handleCarousel($carousel); }
+    else if ($collapsible.length) { handleCollapsible($collapsible); }
+
+    // Apply select2 to any select boxes that dont yet have it
+    $('select').not('.select2-hidden-accessible').select2({ dropdownAutoWidth: true });
```
Comment (TysonRV) [inline jazzmin/static/jazzmin/js/change_form.js:123]: @farridav while testing, I've seen that this is adding a text input on m2m filters, as they also have a hidden select2

Screenshot 2020-10-12 at 16 55 02

**Pull request merged**